### PR TITLE
Update global dictionary with new attributes

### PR DESCRIPTION
**Pull request by douglas-reid**
This PR adds the new attributes proposed in [Updated Source & Destination Attributes](https://docs.google.com/document/d/1IcR6C7j5Vl-4Jx8ervVT4lCTBDrBgsD7Uac77K5C5h4/edit#) to the
global dictionary used for communication with Mixer.

### CODE REVIEW

File: mixer/v1/global_dictionary.yaml [hunk 1]
```diff
@@ -212,14 +212,14 @@
...
 - destination.ip
 - destination.port
-- destination.service
+- destination.service # deprecated, please refer to destination.service.host
 - destination.name
 - destination.uid
 - destination.namespace
 - destination.labels
-- destination.user
+- destination.user # deprecated, please refer to destination.principal
```
Comment (qiwzhang) [inline mixer/v1/global_dictionary.yaml]: Sorry,  currently the script passing this file could not handle adding comment in the same line.   
Please add comment in the separate line.  Unless you fix the script.  
There are two scripts:  

https://github.com/istio/proxy/blob/master/src/istio/mixerclient/create_global_dictionary.py

Comment (douglas-reid) [reply mixer/v1/global_dictionary.yaml]: updated.

**Comment by douglas-reid** [comment]
I've updated this PR to move comments and add two new attributes for destination container info.

PTAL.

This PR is pending design review of the doc mentioned in the PR description.

**Comment by douglas-reid** [comment]
@geeknoid PTAL. I believe this reflects the final status from the update proposal.

**Comment by douglas-reid** [comment]
@geeknoid I'm not able to merge PRs in this repo. Whenever you feel comfortable merging, I would appreciate your help in merging.

**Pull request merged**